bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEPType.java

```diff
@@ -195,6 +196,10 @@
             CHANNEL_WINDOWHANDLESTATE, CHANNEL_CONTACT, CHANNEL_BATTERY_VOLTAGE),
     MechanicalHandle03(RORG._4BS, 0x14, 0x0A, false, A5_14_0A.class, THING_TYPE_MECHANICALHANDLE,
             CHANNEL_WINDOWHANDLESTATE, CHANNEL_CONTACT, CHANNEL_VIBRATION, CHANNEL_BATTERY_VOLTAGE),
+    MechanicalHandle04(RORG.VLD, 0x06, 0x01, false, "Soda", 0x0043, D2_06_01.class, THING_TYPE_MECHANICALHANDLE,
```

Comment:
If it is not neccessary I would not narrow the EEP to manufacturer Soda. This feature is just needed for manufactures who implement an EEP in a different way as normal. This is in most cases just true for elt... (the blue one ;)

Reply:
Since Soda created this EEP specification it should be fine to remove the manufacturer specific code. However since I don't own the device I cannot test this myself. Do you think it is it safe to remove these parameters?